<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signin</class>
 <widget class="QDialog" name="Signin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>360</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>设置管理员密码</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src/icons.qrc">
    <normaloff>:/mainui/icon/MainIcon.ico</normaloff>:/mainui/icon/MainIcon.ico</iconset>
  </property>
  <widget class="QLineEdit" name="le_pswd">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>110</y>
     <width>252</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border-style:none;
	padding:6px;
	border-radius:15px;
	border:2px solid #DCE4EC;
}
QLineEdit:focus {
	border:2px solid #34495E;
}
</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhEmailCharactersOnly</set>
   </property>
   <property name="maxLength">
    <number>256</number>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_proof">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>190</y>
     <width>252</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border-style:none;
	padding:6px;
	border-radius:15px;
	border:2px solid #DCE4EC;
}
QLineEdit:focus {
	border:2px solid #34495E;
}
</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhEmailCharactersOnly</set>
   </property>
   <property name="maxLength">
    <number>256</number>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>431</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>设置管理员密码，用于进入此应用；
每个计算机用户仅允许一个管理员存在;.
密码长度为6~256，可包含字母、数字、符号（@ . # $）</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>421</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pbt_cancel">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-style:none;
    padding:8px;
    border-radius:12px;
    color:#FFFFFF;
    background:#34495E;
    border:2px solid #34495E;
}
QPushButton:hover {
    color:#4E6D8C;
    background:#F0F0F0;
}
QPushButton:pressed {
    color:#2D3E50;
    background:#B8C6D1;
}
</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pbt_setting">
      <property name="font">
       <font>
        <family>Microsoft YaHei UI</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-style:none;
    padding:8px;
    border-radius:12px;
    color:#FFFFFF;
    background:#34495E;
    border:2px solid #34495E;
}
QPushButton:hover {
    color:#4E6D8C;
    background:#F0F0F0;
}
QPushButton:pressed {
    color:#2D3E50;
    background:#B8C6D1;
}
QPushButton:disabled{
	background:rgb(134, 134, 134);
	border-color:rgb(134, 134, 134);
}
</string>
      </property>
      <property name="text">
       <string>设置</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="warning">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>230</y>
     <width>212</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 黑体 Std R</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="src/icons.qrc"/>
 </resources>
 <connections/>
</ui>
